.gitignore

```diff
@@ -5,3 +5,4 @@
 lmutil.exe
 venv/
 .idea/
+app/config.py
```

Comment:
This seems like a core part of the app. Any reason you are telling git to ignore it?